test/.eslintrc

```diff
@@ -0,0 +1,8 @@
+{
+  "globals": {
+    "sinon": false
+  },
+  "rules": {
+    "prefer-arrow-callback": 0
+  }
+}
```

Comment:
Do we want to use nested .eslintrc config files, or use `overrides` section in the root .eslintrc.js file?
https://eslint.org/docs/user-guide/configuring#configuration-based-on-glob-patterns

Also, looking at some of the files below, we may want to override the `env` here and make everything in test/** use the `node` and `mocha` envs, and maybe just disable the `no-console` rule for all test files.

Reply:
And if we prefer a nested test/.eslintrc file, we should probably rename it to test/.eslintrc.js and `module.exports` the config, to be consistent with the root ../.eslintrc.js file.